docs/player_factionchange_items.md

```diff
@@ -97,7 +97,7 @@ Basically all item changes made when player changes faction.
 
 Not implemented.
 
-See item\_template.AllowableRace
+See [item\_template.AllowableRace](http://www.azerothcore.org/wiki/item_template#allowablerace)
 
```

Comment:
On most links, you put a relative link, but on some like this one, it's a direct link. Why?

It can't be a relative link? @Kitzunu

Reply:
I might have missed some...
I made most of these between the hours of 2-4am (yes I know stupid 😂)

If you see anything that is missing please point it out and I will check

Reply:
My intent for all those were to point them directly to the part of the page where it concerns. However mistaken could be made

Reply:
If it's not a relative link, then if we ever change the domain name, then all links will be fucked again :'(